sensitivity.tex

```diff
@@ -108,63 +108,51 @@ \subsection{Lifetime Extension of French \glspl{LWR}}\label{sec:life}
     \label{fig:pow_diff}
 \end{figure}
 
-Figure \ref{fig:ext_uox} shows the decrease in the average monthly
-\gls{UOX} reprocessing burden with increased \gls{LWR} lifetimes.
-This is balanced by extending the necessary duration of 
-\gls{LWR} \gls{UNF} reprocessing,
-so
-the total amount of \gls{LWR} \gls{UNF} reprocessed increases nonetheless.
 
+Figure \ref{fig:ext_fuel} shows the change in \gls{ASTRID} fuel loading with
+\gls{LWR} lifetime extension. The \gls{ASTRID} fuel loaded with plutonium
+from \gls{LWR} \gls{UNF} convey the corresponding \gls{LWR} \gls{UNF} reprocessing demand.
+The change in \gls{ASTRID} deployment alters \gls{ASTRID} fuel loading patterns,
+but does not increase the \gls{LWR} \gls{UNF} demand significantly.
```

Comment:
changed to:
```The change in \gls{ASTRID} deployment alters \gls{ASTRID} fuel loading patterns.
 However, increasing the \gls{LWR} lifetimes
does not increase the \gls{LWR} \gls{UNF} demand significantly (less than 4\%
for the 20-year extension case),
because \glspl{ASTRID} become self-sustaining at similar times after the first
\gls{ASTRID} deployment.```